Loader/Loader.py

```diff
@@ -273,12 +337,18 @@ def initLogging(self, loglevel):
 
     parser = argparse.ArgumentParser(description='Command line parser')
 
-    parser.add_argument('-f', '--fileName',       help='YAML Specification file name', required=True)
-    parser.add_argument('-r', '--dynamoDBRegion', help='Region where the DynamoDB configuration exists.', required=True)
-    parser.add_argument('-v', '--validateOnly', help='Only verify the Yaml file, do not execute any changes', action="store_true", required=False)
-    parser.add_argument('-l', '--logLevel', choices=['critical', 'error', 'warning', 'info', 'debug', 'notset'], help='The level to log', required=False)
+    parser.add_argument('-f', '--fileName',       help='YAML Specification file name', required=False)
+    parser.add_argument('-r', '--dynamoDBRegion', help='Region where the DynamoDB configuration exists.', required=False)
+    parser.add_argument('-v', '--validateOnly',   help='Only verify the Yaml file, do not execute any changes', action="store_true", required=False)
+    parser.add_argument('-l', '--logLevel',       choices=['critical', 'error', 'warning', 'info', 'debug', 'notset'], help='The level to log', required=False)
+
+    parser.add_argument('-s', '--sourceWorkloadPath',help="Provide the relative path of the source workload yaml file e.g  ../../AWS-Scheduling/loadDynamoDB/workloads/Quest-QA09.yaml",required=False)
+    parser.add_argument('-d', '--newWorkloadPath',   help="Provide the relative path of the destination workload yaml file e.g  ../../AWS-Scheduling/loadDynamoDB/workloads/Quest-QA10.yaml",required=False)
+    parser.add_argument('-u', '--unloadFilePath',help="Provide the relative path of the workload yaml file to delete from filesystem and unload from DDB e.g  ../../AWS-Scheduling/loadDynamoDB/workloads/Quest-QA10.yaml",required=False)
 
```

Comment:
I don't know what the purpose is of Source, New, and Unload file paths.  Would need to update the documentation page.